src/peer_manager.rs

```diff
@@ -653,14 +619,8 @@ impl PeerManager {
         trace!("{}No candidate is currently being handled.", log_prefix);
     }
 
-    /// Tries to add the given peer to the routing table. If successful, this returns `Ok(true)` if
-    /// the addition should cause our section to split or `Ok(false)` if the addition shouldn't
-    /// cause a split.
-    pub fn add_to_routing_table(&mut self,
-                                pub_id: &PublicId,
-                                peer_id: &PeerId,
-                                want_to_merge: bool)
-                                -> Result<bool, RoutingTableError> {
+    /// Called just before `RouteManager::add_to_routing_table()`.
+    pub fn pre_add_to_table(&mut self, peer_id: &PeerId) {
```

Comment:
Does `self` need to be mutable here?
If this method doesn't do anything anyway, I'd rather just include these log messages in `set_to_full_node`.